api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -144,34 +147,55 @@ public SortedMap<String, String> getSystemVariables() throws APIException {
 	/**
 	 * @see org.openmrs.api.AdministrationService#getGlobalProperty(java.lang.String)
 	 */
+	
 	@Override
 	@Transactional(readOnly = true)
 	public String getGlobalProperty(String propertyName) throws APIException {
-		// This method should not have any authorization check
 		if (propertyName == null) {
 			return null;
 		}
+		try {
+			Context.addProxyPrivilege(PrivilegeConstants.GET_GLOBAL_PROPERTIES);
+			if (!Context.isAuthenticated()) {
```

Comment:
The authentication check is automatically done by the framework. So not need for all this logic here. Not so?

Reply:
@dkayiwa you must be right. Let me remove it